src/index.html

```diff
@@ -21,7 +24,482 @@
     />
   </head>
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a
+        class="header__picture"
+        href="./images/logo.png"
+      >
+        <img
+          src="./images/logo.png"
+          alt="MOYO-logo"
+        />
```

Comment:
The `href` attribute in the `a` tag should contain a URL where the user will be redirected when they click on the link. Currently, it contains a path to the image file, which is incorrect. You should change it to the URL of your website or some other URL.